src/layouts/ErrorBoundary.js

```diff
@@ -26,6 +28,7 @@ export default class ErrorBoundary extends Component {
 
     if (hasError) {
       const errorId = logError({ error: error, codeLocation: 'codeLocation', type: 'crash' })
+      // @ts-ignore
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142059025-a3d2ae49-c9b0-43c3-a1d3-ce8ded96b2a0.png)

Reply:
This have to be solved, type is clearly wrong.

@Katsu08 please help me adding a ticket to investigate this type error